server/src/test/java/org/solarbank/server/unit/dto/CalculateRequestTest.java

```diff
@@ -0,0 +1,115 @@
+package org.solarbank.server.unit.dto;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.solarbank.server.dto.CalculateRequest;
+import org.solarbank.server.dto.EnergyTariff;
+import org.solarbank.server.dto.Location;
+import org.solarbank.server.dto.PanelSize;
+import org.solarbank.server.ValidationMessage;
+
+import java.util.Set;
+
+import static org.solarbank.server.CreateMockCalculateRequest.createCalculateRequest;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CalculateRequestTest {
+    private Validator validator;
+
+    private CalculateRequest calculateRequest = createCalculateRequest();
```

Comment:
This should be initialised inside the `@BeforeEach`  as a new object for each test because your mutating its state.

Reply:
moved to setUp method